common/app/com/commercetools/sunrise/search/pagination/viewmodels/AbstractPaginationViewModelFactory.java

```diff
@@ -121,11 +121,11 @@ private long calculateTopThresholdPage(final long currentPage, final long totalP
         return Math.min(currentPage + paginationSettings.getDisplayedPages(), totalPages);
     }
 
-    private long calculateTotalPages(final PagedResult<?> pagedResult, final long currentPage) {
+    static long calculateTotalPages(final PagedResult<?> pagedResult, final long currentPage) {
```

Comment:
You had to drop the `private` modifier for testing?

Reply:
Yep